components/tables/ProjectTable/ProjectRow/ProjectRow.tsx

```diff
@@ -0,0 +1,158 @@
+import DeleteProjectModal from "@/components/modals/DeleteProjectModal";
+import SnackbarAlert from "@/components/SnackbarAlert";
+import { PAGES } from "@/helpers/navigation";
+import { Mutate } from "@/hooks/useFetch";
+import useSnackbarAlert from "@/hooks/useSnackbarAlert";
+import { LEVELS_OF_ACHIEVEMENT, Project } from "@/types/projects";
+import { KeyboardArrowDown } from "@mui/icons-material";
+import {
+  Button,
+  Chip,
+  Menu,
+  MenuItem,
+  Stack,
+  TableCell,
+  TableRow,
+} from "@mui/material";
+import Link from "next/link";
+import { FC, MouseEvent, useState } from "react";
+
+type Props = {
+  project: Project;
+  mutate: Mutate<Project[]>;
+};
+
+const ProjectRow: FC<Props> = ({ project, mutate }) => {
+  const { snackbar, setSuccess, setError, handleClose } = useSnackbarAlert();
+  const [dropdownAnchorElement, setDropdownAnchorElement] =
+    useState<HTMLElement | null>(null);
+  const isDropdownOpen = Boolean(dropdownAnchorElement);
+  const [isDeleteProjectOpen, setIsDeleteProjectOpen] = useState(false);
+
+  const handleOpenDropdown = (e: MouseEvent<HTMLButtonElement>) => {
+    setDropdownAnchorElement(e.currentTarget);
+  };
+
+  const handleCloseDropdown = () => {
+    setDropdownAnchorElement(null);
+  };
+
+  const handleOpenDeleteModal = () => {
+    setIsDeleteProjectOpen(true);
+  };
+
+  const renderTags = () => {
```

Comment:
This shouldn't return an array of tags, just one tag because each project can only have 1 LOA at anytime